src/mds/Server.cc

```diff
@@ -462,9 +464,12 @@ void Server::reclaim_session(Session *session, const cref_t<MClientReclaim> &m)
   if (flags & CEPH_RECLAIM_RESET) {
     finish_reclaim_session(session, reply);
     return;
```

Comment:
unneeded return since you have a `else` block.

Reply:
removed, thanks!